neqo-transport/src/recv_stream.rs

```diff
@@ -33,6 +33,21 @@ pub const RECV_BUFFER_SIZE: usize = RX_STREAM_DATA_WINDOW as usize;
 
 pub(crate) type RecvStreams = BTreeMap<StreamId, RecvStream>;
```

Comment:
```suggestion
pub(crate) struct RecvStreams(BTreeMap<StreamId, RecvStream>);
```

I know that this means a bunch more factoring, so I'd be OK with a separate change, but it would be a good change.